lib/transbank/sdk/webpay/webpay_plus/mall/mall_transaction.rb

```diff
@@ -0,0 +1,89 @@
+module Transbank
+  module Webpay
+    module WebpayPlus
+      class MallTransaction
+        extend Transbank::Utils::NetHelper
+        CREATE_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+        COMMIT_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+        TRANSACTION_STATUS_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions/:token'
+        REFUND_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions/:token/refunds'
+        class << self
+
+          def create(buy_order:, session_id:, return_url:, details:, options: nil)
+            api_key = options&.api_key || default_integration_params[:api_key]
+            commerce_code = options&.commerce_code || default_integration_params[:commerce_code]
+            integration_type = options&.integration_type || default_integration_params[:integration_type]
+            base_url = integration_type.nil? ? WebpayPlus::Base::integration_type[:TEST] : WebpayPlus::Base.integration_type_url(integration_type)
+
+            url = base_url + CREATE_TRANSACTION_ENDPOINT
+            headers = webpay_headers(commerce_code: commerce_code, api_key: api_key)
+            body = {buy_order: buy_order,
+                    session_id: session_id,
+                    details: details,
+                    return_url: return_url}
+            resp = http_post(uri_string: url, body: body, headers: headers, camel_case_keys: false)
+            json = JSON.parse(resp.body)
```

Comment:
This might throw a parse error. Are you sure you don't want to check first for http status code?

Reply:
Maybe it'd be a good idea to check if the response is `#kind_of? Net::HTTPSuccess` and afterwards check that the body is not empty (since `JSON.parse` raises when trying to parse an empty string)